### PR TITLE
Add Hiring role definition 

### PR DESCRIPTION
**Pull request by alexeh**

### CODE REVIEW

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+# Capacity role definition
+
+**Current responsible**: -
+
+
+
+***Note:***
+
+2 people Identify the hiring related tasks to be managed between different roles:
```
Comment (agnessa) [inline _docs/backenders/strategy-2021/roles/hiring.md:16]: Perhaps all these roles should be involved in the advice process on hiring. Might need adding to the "decision making" for these roles' descriptions. https://github.com/Vizzuality/devismos/pull/14 https://github.com/Vizzuality/devismos/pull/15 https://github.com/Vizzuality/devismos/pull/16

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- Preparing candidate profile descriptions for outsourcing needs
+  
+          Priority: Medium ( 3 Medium, 1 High) 
+          Identified as responsability to be shared with Capacity Role
+          Andr's note: This responsibility should me managed by each developer, with advice from at least another team member
+                       We should try to keep this rather standard, once we can get there of course
+
+
+## Impact
+
```
Comment (agnessa) [inline _docs/backenders/strategy-2021/roles/hiring.md:64]: Reduce administrative burden on the BE team
Improve timeliness of starting hiring processes

Comment (alexeh) [reply _docs/backenders/strategy-2021/roles/hiring.md:64]: Added 88912f620275e49546b134901c22eb14d9283e3f
Thanks 😃

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+## Responsibilities
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
+          
+          Priority: HIGH priority for all   
+          Identified as responsibility to be shared with Capacity Role 
+
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
```
Comment (tiagojsag) [inline _docs/backenders/strategy-2021/roles/hiring.md:29]: is this repeated from line 23?

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions 
+  
+          Priority: HIGH priority for all   
+          Identified as responsibility to be shared with Capacity, Strategy, M.C roles
+
+
+- Distributing tasks: reviewing candidates, attending interviews
+        
+          Priority: Average Medium priority. (2 Low, 1 Medium, 1 High)
+          Agn and Andr are happy with the way that Sebastion is managing his part of this responsibility
```
Comment (tiagojsag) [inline _docs/backenders/strategy-2021/roles/hiring.md:38]: keyboards are cheap, no need to save on characters typed :P please use full names

**Review by tiagojsag — CHANGES_REQUESTED** [review]
I disagree with adding notes from people, and also identifying levels of priority, to some extent. IMO this role definition should describe the role itself, not the decision process we went through to get there. In light of that, it's not important who says what, and instead you should focus on what the role owner (you) is committing to do as part of this role.

**Review by agnessa — APPROVED** [review]
Since we've decided the hiring role is not needed at this point, we have two ways to go about this PR:
- close it
- merge it, but list the role under "draft" roles or sth like that, i.e. it will be there ready as a draft if at some point we decide we need it. I wouldn't put more work into it at this time though.

**Review by aagm — APPROVED** [review]
My only concern and maybe that will involves Strategy(@agnessa) will be to have in mind a long-term growth plan (in terms of hiring) for the team. Also, we don't have someone to take care of this as a representative. shall we left the pr open or merge it, and maybe attached it as a sub-responsability for capacity?

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+          Andr thinks this responsibility should me managed by each developer
```
Comment (ikas) [inline _docs/backenders/strategy-2021/roles/hiring.md:39]: I don't think we should refer to BE members explicitly, this document should be independent of the current BE team members. I'd suggest phrasing these opinions ignoring who said them.

**Review by ikas — CHANGES_REQUESTED** [review]
I'd suggest removing all references to BE team members - this document should be independent from the individual team members IMO.

Also, inner blocks in the responsibilities sections are being formatted like code snippets on the lines that start with tabs - I'd suggest formatting them as sub-lists :)

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Template
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Hiring role definition -- *draft*
```
Comment (tiagojsag) [inline _docs/backenders/strategy-2021/roles/hiring.md:8]: we should remove the "draft" and name the responsible person.

**Review by tiagojsag — CHANGES_REQUESTED** [review]

File: _docs/backenders/strategy-2021/roles/hiring.md (hunk 6 above)
Comment (hotzevzl) [reply _docs/backenders/strategy-2021/roles/hiring.md:8]: I think nobody wanted to take this role and somehow it ended up being weighted as lower priority.

As Agnieszka commented in the PR thread, we should either close this PR and shelve it, or keep it as draft with no responsible person. In practice I think I have been taking care (in a very lightweight fashion) of some of these tasks and in general I don't mind keep bringing up the "let's march on with hiring" mantra all the time, and except for some revisions to the job ad text the level of effort is normally negligible, so I'd even be happy to keep bugging the team with "let's march on with hiring" 🤷🏼

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
+  
+    - Identified as responsibility to be shared with Capacity Role 
+
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
+          
+    - Identified as responsibility to be shared with Capacity, Strategy, M.C roles 
+
```
Comment (hotzevzl) [inline _docs/backenders/strategy-2021/roles/hiring.md:22]:
```suggestion
- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
          
  *Responsibility to be shared with Capacity, Strategy and M.C roles*
```

Comment (hotzevzl) [reply _docs/backenders/strategy-2021/roles/hiring.md:22]: what is the MC role though? 🤔

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
+          
+    - Identified as responsibility to be shared with Capacity, Strategy, M.C roles 
+    
+    
+- Distributing tasks: reviewing candidates, attending interviews
+
+
```
Comment (hotzevzl) [inline _docs/backenders/strategy-2021/roles/hiring.md:26]:
```suggestion

```

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+- Liaising with P&C about onboarding of new BE members
+  
+    - Identified as responsibility to be shared with Capacity, Strategy, M.C roles
+
```
Comment (hotzevzl) [inline _docs/backenders/strategy-2021/roles/hiring.md:30]:
```suggestion
    *Responsibility to be shared with Capacity, Strategy and M.C roles*
```

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+- Liaising with P&C about onboarding of new BE members
+  
+    - Identified as responsibility to be shared with Capacity, Strategy, M.C roles
+    
+    
+- Collaborating with P&C on job ads text
+  
+    - Identified as responsibility to be shared with Capacity, Strategy roles
+
```
Comment (hotzevzl) [inline _docs/backenders/strategy-2021/roles/hiring.md:35]:
```suggestion
    *Responsibility to be shared with Capacity and Strategy roles*
```

File: _docs/backenders/strategy-2021/roles/hiring.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+- Collaborating with P&C on job ads text
+  
+    - Identified as responsibility to be shared with Capacity, Strategy roles
+    
+    
+- Preparing candidate profile descriptions for outsourcing needs
+
+   - Identified as responsibility to be shared with Capacity Role
+
+
```
Comment (hotzevzl) [inline _docs/backenders/strategy-2021/roles/hiring.md:41]:
```suggestion
- Preparing candidate profile descriptions for outsourcing needs

  *Responsibility to be shared with Capacity role*
```

Comment (hotzevzl) [reply _docs/backenders/strategy-2021/roles/hiring.md:41]: I think in practice these profiles will mostly be better handled by project developers with input from PMs, but I don't see any harm in keeping this here (especially if nobody is taking this role right now 😁)

**Review by hotzevzl — APPROVED** [review]

File: _docs/backenders/strategy-2021/roles/hiring.md (hunk 7 above)
Comment (alexeh) [reply _docs/backenders/strategy-2021/roles/hiring.md:22]: Mentorship Coordination

**Pull request merged**